internal/dev_server/ui/asset_handler.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 )
 
-//go:embed index.html
+//go:embed dist/index.html
```

Comment:
@mike-zorn the output of Vite by default is `{repo_root}/internal/dev_server/ui/dist/*` because we run the build command from `{repo_root}/internal/dev_server/ui`... as a result I had to change this embed pattern, and when browsing to `localhost:8765/ui` I was present with a list of filesystem contents (guessing the static file serving middleware?)

..browsing to `localhost:8765/ui/dist` results in the `{repo_root}/internal/dev_server/ui/dist/index.html` file being served as expected, but I don't think we want the `"dist"` portion of the path? Not a big deal, just noting for posterity

Reply:
We also made the decision _not_ to introduce the frontend build into the build pipeline at this point, because we'll need to update the GH Action to use node/npm to run `npm run build`.

...for now, let's just run `npm run build` locally and check in the artifact.

Reply:
Took a look. I think I see how to fix it. I'll add a commit to the PR.

Reply:
Pulled the change down and tested, looks good. Thanks! 🙌